torchgeo/datasets/globbiomass.py

```diff
@@ -0,0 +1,282 @@
+        "N80W140_agb.zip": "7719c0efd23cd86215fea0285fd0ea4a",
+        "N80W140_gsv.zip": "499969bed381197ee9427a2e3f455a2e",
+        "N80W180_agb.zip": "e3a163d1944e1989a07225d262a01c6f",
+        "N80W180_gsv.zip": "5d39ec0368cfe63c40c66d61ae07f577",
+        "S40E140_agb.zip": "263eb077a984117b41cc7cfa0c32915b",
+        "S40E140_gsv.zip": "e0ffad85fbade4fb711cc5b3c7543898",
+        "S40W060_agb.zip": "2cbf6858c48f36add896db660826829b",
+        "S40W060_gsv.zip": "04dbfd4aca0bd2a2a7d8f563c8659252",
+        "S40W100_agb.zip": "ae89f021e7d9c2afea433878f77d1dd6",
+        "S40W100_gsv.zip": "b6aa3f276e1b51dade803a71df2acde6",
+    }
+
+    def __init__(
+        self,
+        root: str = "data",
+        crs: Optional[CRS] = None,
+        res: Optional[float] = None,
+        measurement: str = "agb",
+        transforms: Optional[Callable[[Dict[str, Any]], Dict[str, Any]]] = None,
+        cache: bool = True,
+        checksum: bool = False,
+    ) -> None:
+        """Initialize a new Dataset instance.
+
+        Args:
+            root: root directory where dataset can be found
+            crs: :term:`coordinate reference system (CRS)` to warp to
+                (defaults to the CRS of the first file found)
+            res: resolution of the dataset in units of CRS
+                (defaults to the resolution of the first file found)
+            measurement: use data from 'agb' or 'gsv' measurement
+            transforms: a function/transform that takes an input sample
+                and returns a transformed version
+            cache: if True, cache file handle to speed up repeated sampling
+            checksum: if True, check the MD5 of the downloaded files (may be slow)
+
+        Raises:
+            FileNotFoundError: if no files are found in ``root``
+            RuntimeError: if dataset is missing or checksum fails
+            AssertionError: if measurement argument is invalid, or not a str
+        """
+        self.root = root
+        self.checksum = checksum
+
+        assert isinstance(measurement, str), "Measurement argument must be a str."
+        assert (
+            measurement in self.measurements
+        ), "You have entered an invalid measurement, please choose one of {}.".format(
+            self.measurements
+        )
+        self.measurement = measurement
+
+        self.filename_glob = "*0_{}*[!zip]".format(self.measurement)
+        self.zipfile_glob = "*0_{}.zip".format(self.measurement)
+
+        self._verify()
+
+        super().__init__(root, crs, res, transforms, cache)
+
+    def __getitem__(self, query: BoundingBox) -> Dict[str, Any]:
+        """Retrieve image/mask and metadata indexed by query.
+
+        Args:
+            query: (minx, maxx, miny, maxy, mint, maxt) coordinates to index
+
+        Returns:
+            sample at index consisting of measurement mask with key 'mask',
+            standard error under 'error_mask' key, crs under 'crs' and
```

Comment:
I wonder if we should concatenate gsv, agb, and agb_err into a single 3-channel tensor and return that instead.

Reply:
So there is an error map for each measurement: so gsv and gsv_err, as well as agb and agb_err. Therefore, either would have to concatenate to a 4-channel tensor or get a 2-channel tensor per measurement.

Reply:
I think we should definitely try to use a single 2-channel or 4-channel tensor, we don't want to use custom keys for every dataset in TorchGeo. Are GSV and AGB separate downloads? If so, I'm fine with keeping the "measurement" parameter and making this a single 2-channel tensor. If everything is always bundled, then we can make it a 4-channel tensor.

Reply:
yes, gsv and agb are separate downloads. I will make a 2-channel  tensor.